go/inst/instance_dao.go

```diff
@@ -429,7 +429,7 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
 						instance.SemiSyncMasterEnabled = (m.GetString("Value") == "ON")
 						semiSyncMasterPluginLoaded = true
 					} else if m.GetString("Variable_name") == "rpl_semi_sync_master_timeout" {
-						instance.SemiSyncMasterTimeout = m.GetUint("Value")
+						instance.SemiSyncMasterTimeout = m.GetInt64("Value")
```

Comment:
Don;t we need to support max UINT64?
```suggestion
						instance.SemiSyncMasterTimeout = m.GetUint64("Value")
```

Reply:
there is no `GetUint64` in https://github.com/openark/golib/blob/master/sqlutils/sqlutils.go